charts/generic-govuk-app/templates/dbmigration-job.yaml

```diff
@@ -9,6 +9,7 @@ metadata:
     app: {{ $fullName }}-dbmigrate
     app.kubernetes.io/name: {{ $fullName }}-dbmigrate
     app.kubernetes.io/component: dbmigrate
+    app.kubernetes.io/arch: {{ .Values.arch }}
```

Comment:
You might be able to put this in generic-govuk-app.labels in _helpers.tpl and cover all the various places in one go.